test/expint.jl

```diff
@@ -201,7 +201,7 @@ using Base.MathConstants
     @test expint(-2, -2.2) ≈ -2.0680909972407264331884989 rtol=1e-14
     @test expint(-2.2, 3.2) ≈ 0.024950173497409329191241353395358 rtol=1e-14
     @test expint(+2.2, 3.2) ≈ 0.008044603700773423319087602010 rtol=1e-12
-    @test expint(3.2, 1.3) ≈ 0.070147692224611216675759479422283315452559216337905 rtol=1e-14
+    @test expint(3.2, 1.3) ≈ 0.070147692224611216675759479422283315452559216337905 rtol=1.1e-14
```

Comment:
This is needed to make tests pass on Alpine Linux

Reply:
Can you leave a comment to that effect? It's weird that this one has a different `rtol` than all of the ones surrounding it.

Reply:
I could also increase to 1e13 and it would look less oddly specific.  These aren't all 1e-14, there are other tests with precision of 1e-12